.github/workflows/cuda.yml

```diff
@@ -110,7 +110,7 @@ jobs:
         which nvcc || echo "nvcc not in PATH!"
 
         git clone https://github.com/AMReX-Codes/amrex.git ../amrex
-        cd ../amrex && git checkout --detach 6d30e83c944e4e6167178f2d145df2a3e67d2b24 && cd -
+        cd ../amrex && git checkout --detach 5f8030af0ddfe99139968bcf896b5662cccf94a4 && cd -
```

Comment:
```suggestion
        cd ../amrex && git checkout --detach 6d30e83c944e4e6167178f2d145df2a3e67d2b24 && cd -
```

Reply:
This need to be here - this PR uses functions from AMReX that don't exist in version `6d30e83c944e4e6167178f2d145df2a3e67d2b24`

Reply:
Oh, yes - sorry for this. I forgot that #3772 was not yet in.

 #3788 will fix this in the next weekly update.